pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
+requires = ["pbr>=6.0.0", "setuptools>=64.0.0"]
+build-backend = "pbr.build"
+
+[project]
+requires-python = ">=3.8"
```

Comment:
I think we should say 3.10, Jupyter is using this at the moment, and that is typically our source of truth for is python-chi working.